Common Programming Concepts/section-info.yaml

```diff
@@ -2,5 +2,5 @@ content:
 - Variables
 - Data Types
 - Functions
-- Variables and Functions Test
+- "Variables, Functions and If Test"
```

Comment:
Does this attach the lesson correctly? I get an error with the quotes, but it seems to work fine without them/